### PR TITLE
Updating the delete agent functionality.

### LINKED ISSUE #6: To implement the function to update and delete agent
Definition of done:

Provide an option for admin to update and delete the agent.

### PR DESCRIPTION
**Pull request by SuyesSapkota**

### CODE REVIEW

File: food/views.py [hunk 1]
```diff
@@ -322,19 +322,43 @@ def EditProfile(request, d_id):
...
     except Agent_Model.DoesNotExist:
+        # If no instance is found, do nothing (no error raised, no deletion).
         pass
 
+    # Delete the Agent_Model instance (if it exists).
     a_sel.delete()
-    return redirect('ViewAgent')
+
+    # Redirect the user to the 'ViewSDonars' page.
+    return redirect('ViewSDonars')
```
Comment (Temporary-summer) [inline food/views.py]: Here I guess it should redirect to 'ViewAgent' page but it is redirecting to 'ViewSDonars' page once please check it.

Comment (SuyesSapkota) [reply food/views.py]: Yes I will look into it.

**Comment by SuyesSapkota** [comment]
I have made the changes. Can you please verify it ?

**Review by SuyesSapkota — COMMENTED** [review]
.

**Review by Temporary-summer — APPROVED** [review]
I can see it is redirecting me to the ViewAgent page now and I can successfully delete the agent. It can be merged now.

**Pull request merged**